senza/cli.py

```diff
@@ -417,11 +417,16 @@ def create(definition, region, version, parameter, disable_rollback, dry_run, fo
     for name, parameter in data.get("Parameters", {}).items():
         parameters.append([name, getattr(args, name, None)])
 
-    tags = {
+    tags = {}
+    for tag in input["SenzaInfo"].get('Tags', ()):
```

Comment:
This will fail if "Tags" is missing (your are using an empty tuple)
Please add a unit test for this code path.